packages/flutter/lib/fix_data.yaml

```diff
@@ -15,6 +15,18 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/84467
+  - title: "Migrate to 'detailedControlsBuilder'"
+    date: 2021-07-22
+    element:
+      uris: [ 'material.dart' ]
+      constructor: ''
+      inClass: 'Stepper'
+    changes:
+      - kind: 'renameParameter'
+        oldName: 'controlsBuilder',
```

Comment:
```suggestion
        oldName: 'controlsBuilder'
```

Reply:
😜

Reply:
You'll also want to add one for the field member. Search `resizeToAvoidBottomInset` in `fix_data.yaml` for a good example. Same for the tests in `test_fixes/material.dart` and `test_fixes/material.dart.expect` :)